src/styles/global-styles.ts

```diff
@@ -10,7 +10,8 @@ export const globalStyles = css`
         margin: 0;
         padding: 0;
         background-color: var(--background);
-        overflow: hidden;
+        overflow-x: hidden;
+        overflow-y: visible;
```

Comment:
I still don't understand why this change is needed

Reply:
This is overflow protection. If our content goes beyond 100vh, then vertical scrolling will be added.